docker/server/config/config-mysql.properties

```diff
@@ -12,12 +12,20 @@ spring.datasource.password=conductor
 spring.datasource.hikari.maximum-pool-size=10
 spring.datasource.hikari.minimum-idle=2
 
+# disabled: it causes error after second docker-compose up
 # Elastic search instance indexing is enabled.
-conductor.indexing.enabled=true
+# conductor.indexing.enabled=true
+
+# Elastic search version (6 || 7)
+conductor.elasticsearch.version=6
```

Comment:
`Elastic search 7` is not seamlessly supported. The comment implies that users could change the value of this property from `6` to `7` and the docker compose would work. But that's not the case. Please remove this comment.